panda/src/audiotraits/openalAudioManager.cxx

```diff
@@ -117,10 +118,11 @@ OpenALAudioManager() {
 
   // Initialization
   if (_active_managers == 0 || !_openal_active) {
-    _device = alcOpenDevice(NULL); // select the "preferred device"
+    string dev_name = get_audio_device();
+    _device = alcOpenDevice(dev_name.c_str()); // select the user or preferred device
```

Comment:
I'm sure it does.  However, there is more than one OpenAL implementation out there, and the OpenAL specification states that NULL should be passed to get the default device.  I'd prefer not risking a change in default behaviour in a minor release of Panda3D.